holoviews/element/util.py

```diff
@@ -282,3 +282,21 @@ def connect_edges(graph):
         end = end_ds.array(end_ds.kdims[:2])
         paths.append(np.array([start[0], end[0]]))
     return paths
+
+
+def validate_sampling(img, dimension, rtol=10e-9):
```

Comment:
Not worth holding up the PR unless you agree, but I think ``check_regularly_sampled`` might be a better name...

Reply:
Or ``check_regular_sampling``

Reply:
I'll go with a mixture, ``validate_regular_sampling``